validator/build.gradle

```diff
@@ -1,97 +1,124 @@
 plugins {
-    id 'java'
+	id 'java'
 
-    id 'application'
-    id 'com.google.cloud.tools.jib' version "3.3.2"
+	id 'application'
+	id 'com.google.cloud.tools.jib' version "3.3.2"
 
-    // lombok
-    id "io.freefair.lombok" version "8.0.1"
+	// lombok
+	id "io.freefair.lombok" version "8.0.1"
+	id("com.diffplug.spotless") version "6.19.0"
 
-    // check style
-    id 'checkstyle'
 }
 
 tasks.withType(Checkstyle) {
```

Comment:
can we remove this if checkstyle is not being used anymore?

Reply:
done, good catch